src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -49,9 +49,9 @@ export class InputTimePicker
   //--------------------------------------------------------------------------
 
   /**
-   * The active state of the time input
+   * When true, the component is active.
    *
-   * @deprecated Use open instead.
+   * @deprecated Use "open" instead.
```

Comment:
Should we use back ticks for all code references or is there a reason why slot doc uses back ticks for component names and props use quotes?

Reply:
Good question! We decided component names should use backticks, whereas props should use quotes per [style docs](https://github.com/Esri/calcite-components/issues/4442#issue-1212723613).

> Use double quotes for property/attribute names and values.

@jcfranco Does this seem like a good approach, or should we revisit adding backticks in to the props as well?

Reply:
Sorry for the delay. I think using backticks would make it easier to scan the doc for API members since it will be syntax highlighted. cc @benelan

Reply:
Based on our conversation and for consistency with other docs site, such as the JSAPI, we will incorporate backticks for prop, slot, and event descriptions. 👍🏻

Reply:
I had wanted to use backticks for prop/slot/event/etc names, and then double quotes for property values when there are union types, e.g. [Accordion's `selectionMode` property](https://github.com/Esri/calcite-components/blob/35b8920516df393a2a80b6593157ddb149c344d1/src/components/accordion/accordion.tsx#L41-L42). That's what I put in the ["official" style guide](https://github.com/Esri/calcite-components/blob/master/conventions/Documentation.md#style-guide). Sorry I guess the one in the issue was outdated.

Reply:
Updated the style guide for consistency, in line with our conventions docs. Changes incoming soon!